server/Review_filter.py

```diff
@@ -1,10 +1,7 @@
 import json, pymysql
 
 # Connect to MySQL and Generete dictionary cursor from connection
-file_reader = open('../db_connection_info', 'r')
-db_connection_info = file_reader.read()
-file_reader.close()
-sql_conn = pymysql.connect(db_connection_info)
+sql_conn = pymysql.connect(host='localhost',user='gaeul',password='alpha',db='CAFE',charset='utf8mb4')

```

Comment:
Do you think it's safe to hardcode the user and password here? Github is public, so pretty much anyone can see this.

Reply:
I just try to hide it but failed. I have to do it. Thank you